test/globalHooks.js

```diff
@@ -19,11 +19,12 @@ before('global setup', () => {
 
   nock('https://api.postcodes.io')
     .get(`/postcodes/${ls27ue}`)
-    .times(4)
+    .times(5)
     .reply(200, ls27ueResponse);
 });
 
 after('Check all nocks have been called', () => {
+  console.log(nock.pendingMocks());
```

Comment:
Yeah, I thought it was useful to have the output easily accessed when the tests were running.